### ISSUE TITLE
Should we use OpenJDK by default?

### ISSUE THREAD

**Issue by sevein** [artefactual-labs/ansible-elasticsearch#3]
openjdk-jre-7-headless is available in 14.04
openjdk-jre-{8,9}-headless available in 16.04

**Comment by sevein** [comment]
https://www.elastic.co/guide/en/elasticsearch/guide/current/_java_virtual_machine.html

**Comment by sevein** [comment]
It seems possible to make this role work with OpenJDK without changes:

```
elasticsearch_version: "1.7.5"
elasticsearch_apt_repos: []
elasticsearch_apt_java_package: "openjdk-7-jre-headless"
elasticsearch_apt_dependencies: []
elasticsearch_java_home: "/usr/lib/jvm/java-7-openjdk-amd64"
elasticsearch_heap_size: "1024m"
```

So I guess the question is... should we make this the default? Low priority.

**Comment by sevein** [comment]
Fixed in #6.
